src/pages/TrueSightV2/constants/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { ChainId } from '@kyberswap/ks-sdk-core'
+
+import { NETWORKS_INFO } from 'constants/networks'
+
+import { KyberAIListType } from '../types'
+
+export const SUPPORTED_NETWORK_KYBERAI: { [chainId in ChainId]?: string } = {
+  [ChainId.MAINNET]: 'ethereum',
+  [ChainId.BSCMAINNET]: 'bsc',
+  [ChainId.AVAXMAINNET]: 'avalanche',
+  [ChainId.MATIC]: 'polygon',
+  [ChainId.FANTOM]: 'fantom',
+  [ChainId.ARBITRUM]: 'arbitrum',
+  [ChainId.OPTIMISM]: 'optimism',
+}

```

Comment:
we should make it an array of supported network in `constants/network.ts` file 😄

Reply:
this one has another purpose is to map between the chain ID and API's chain name.